Sources/StreamChat/Utils/EventBatcher.swift

```diff
@@ -43,7 +44,7 @@ final class Batcher<Item> {
     init(
         period: TimeInterval,
         timerType: Timer.Type = DefaultTimer.self,
-        handler: @escaping ([Item]) -> Void
+        handler: @escaping (_ batch: [Item], _ completion: @escaping () -> Void) -> Void
```

Comment:
```suggestion
        handler: @escaping BatchHandler
```

I'm actually not sure if this (👆) compiles 🤔

Reply:
Yeah, it does not, see the comment above